core/src/main/java/org/vertexium/metric/DropWizardMetricRegistry.java

```diff
@@ -25,12 +26,43 @@ public DropWizardMetricRegistry() {
 
     public DropWizardMetricRegistry(MetricRegistry metricRegistry) {
         this.metricRegistry = metricRegistry;
+        startConsoleReporterUsingSystemProperty();
+    }
+
+    private void startConsoleReporterUsingSystemProperty() {
+        String startConsoleReporter = System.getProperty(START_CONSOLE_REPORTER_PROPERTY_NAME, "false").trim();
+        if (startConsoleReporter.equalsIgnoreCase("false")) {
+            return;
+        }
+
+        if (startConsoleReporter.equalsIgnoreCase("true") || startConsoleReporter.length() == 0) {
+            startConsoleReporter(10, TimeUnit.SECONDS);
+        }
+
+        startConsoleReporter(startConsoleReporter);
     }
 
     public MetricRegistry getMetricRegistry() {
         return metricRegistry;
     }
 
+    private void startConsoleReporter(String period) {
+        try {
```

Comment:
Consider switching this to [support Java duration strings](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#parse-java.lang.CharSequence-).